software/riscv-tools/README.md

```diff
@@ -1,3 +1,18 @@
-Typically you will do
-make checkout-all 
-make build-all
+Clean installation of tools:
```

Comment:
I think this is really helpful!

One thing you can also do is put a `help` rule in the top-level makefile and make it the default rule. That way, when people run 'make`, they are automatically given the help prompt. 

Just a thought - you can do it if you'd like, or keep it this way.